psc-ide.el

```diff
@@ -61,13 +61,13 @@
   :prefix "psc-ide-"
   :group 'psc-ide)
 
-(defcustom psc-ide-server-executable "psc-ide-server"
-  "Path to the 'psc-ide-server' executable."
+(defcustom purs-executable "purs"
```

Comment:
All the variables and functions in this package are prefixed with `psc-ide-`, it would be bad style to have just this variable different. What do you think of `psc-ide-executable` here?